### ISSUE TITLE
EV3Lejos MathConst sqrt(1/2) produces error

### ISSUE THREAD

**Issue by boonto** [OpenRoberta/robertalab#1013]
**Describe the bug**
 - EV3Lejos MathConst sqrt(1/2) produces error.

**To Reproduce**
1. Add a MathConst sqrt(1/2) block to the program.
2. Try to run the program on the robot.

**Expected behavior**
 - The program works and runs on the robot.

**Comment by boonto** [comment]
We currently use BlocklyMethods for most of these math consts:
```java
@Override
public Void visitMathConst(MathConst<Void> mathConst) {
    switch ( mathConst.getMathConst() ) {
        case PI:
            this.sb.append("BlocklyMethods.PI");
            break;
        case E:
            this.sb.append("BlocklyMethods.E");
            break;
        case GOLDEN_RATIO:
            this.sb.append("BlocklyMethods.GOLDEN_RATIO");
            break;
        case SQRT2:
            this.sb.append("BlocklyMethods.sqrt(2)");
            break;
        case SQRT1_2:
            this.sb.append("BlocklyMethods.sqrt(1.0/2.0)");
            break;
        case INFINITY:
            this.sb.append("Float.POSITIVE_INFINITY");
            break;
        default:
            break;
    }
    return null;
}
```
With the Java list revision in #1012 removing BlocklyMethods, we should probably also replace these with pure Java calls.

**Comment by boonto** [comment]
This issue should be fixed in the course of #1016.

**Comment by boonto** [comment]
Fixed for now, a non BlocklyMethods (and probably a *real* constant) solution will be provided in #1016.
